Bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PsrHttpMessage\Bundle\DependencyInjection;
+
+use Laminas\Diactoros\RequestFactory;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Alexander M. Turek <me@derrabus.de>
+ */
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('psr_http_message');
+        $rootNode = $treeBuilder->getRootNode();
+
+        $rootNode->children()
+            ->arrayNode('message_factories')
+                ->{class_exists(Psr17Factory::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                ->children()
+                    ->scalarNode('implementation')
+                        ->info('The PSR-7 implementation to use. By default, the bundle will configure the Nyholm implementation if it\'s available.')
+                        ->defaultValue('nyholm')
+                        ->validate()
+                            ->ifNotInArray(['nyholm', 'diactoros'])->thenInvalid('Supported implementations are: "nyholm", "diactoros".')
+                            ->ifTrue(function ($value) {
+                                return 'nyholm' === $value && !class_exists(Psr17Factory::class);
+                            })->thenInvalid('Cannot configure Nyholm\'s PSR-7 implementation: Try running "composer nyholm/psr7".')
+                            ->ifTrue(function ($value) {
```

Comment:
This is broken. You need to use multiple `->validate()` call if you want to run multiple validation rules. Here, you are replacement the previous if condition with that one.

Reply:
Done, thanks.